addons/l10n_nl/migrations/3.3/post-migrate_update_taxes.py

```diff
@@ -23,6 +23,8 @@ def migrate(cr, version):
     services_taxes = env['account.tax'].browse(_get_tax_ids_for_xml_id(cr, 'btw_X0_diensten'))
 
     old_tax_tags = env['account.account.tag']._get_tax_tags('3bl (omzet)', 'nl')
+    if not old_tax_tags:
+        return
```

Comment:
nitpick: please add a newline after this, otherwise it gets a bit hard to read in my opinion

Reply:
Hello @aliyatastemirova 
Suggested changes have been done.